app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -32,6 +32,7 @@ def needs_proof_of_address?
       attr_reader :enrollment
 
       def due_date
+        enrollment.enrollment_established_at ? enrollment.enrollment_established_at + IdentityConfig.store.in_person_enrollment_validity_in_days.days :
         enrollment.created_at + IdentityConfig.store.in_person_enrollment_validity_in_days.days
```

Comment:
```suggestion
        start_date = enrollment.enrollment_established_at.presence || enrollment.created_at
        start_date + IdentityConfig.store.in_person_enrollment_validity_in_days.days
```
I think it's more readable to use `presence` here, which returns the value or `nil` if it's not defined

Reply:
that's def easier to read!